pkgs/tools/security/steamguard-cli/default.nix

```
@@ -0,0 +1,27 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "steamguard-cli";
+  version = "0.11.0";
+
+  src = fetchFromGitHub {
+    owner = "dyc3";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-xJMz9gKSwjTHdytvu/K8NavzIGeD0nXqg1JMpLmSgmU=";
+  };
+
+  cargoHash = "sha256-LR5ui4rNeIh5BwToBIHGvbLmVz/GzQwVGZak+YDr+i4=";
+
+  meta = with lib; {
+    description = "A linux utility for generating 2FA codes for Steam and managing Steam trade confirmations.";
+    homepage = "https://github.com/dyc3/steamguard-cli";
+    changelog = "https://github.com/dyc3/steamguard-cli/releases/tag/v${version}";
+    license = with licenses; [ gpl3Only ];
+    maintainers = with maintainers; [ surfaceflinger ];
+    mainProgram = "steamguard";
```

```suggestion
    mainProgram = "steamguard";
    platforms = platforms.linux;
```

Don't forget to set platform to linux.
steamguard-cli is not compatible with MacOS

---

I'll fix soon, replying from my phone :) I assumed that because it's a "generic" rust package then it would just work there too :(

Is the proper practice to just set everything I init to linux and wait until someone eventually tests it on darwin?